src/current/_includes/releases/v23.1/v23.1.3.md

```diff
@@ -0,0 +1,147 @@
+    - Ensure applications using the cluster have properly configured [connection pools](../v23.1/connection-pooling.html).
+    - Add more vCPU or more nodes to the cluster.
+    - Increase the password hashing concurrency using the `COCKROACH_MAX_PW_HASH_COMPUTE_CONCURRENCY` [environment variable](../v23.1/cockroach-commands.html#environment-variables). [#104388][#104388]
+
+<h3 id="v23-1-3-db-console-changes">DB Console changes</h3>
+
+- Added merge queue failure and merge queue processing time metrics to the [**Queue Processing Failures**](../v23.1/ui-queues-dashboard.html#queue-processing-failures) and [**Queue Processing Times**](../v23.1/ui-queues-dashboard.html#queue-processing-times) graphs respectively. [#104034][#104034]
+- Removed the **Circuit Breaker Tripped Events** graph from the [**Replication**](../v23.1/ui-replication-dashboard.html) dashboard. [#104038][#104038]
+- Added the **Completed time** field on the [**Jobs**](../v23.1/ui-jobs-page.html) and [**Job details**](../v23.1/ui-jobs-page.html#job-details) pages. Also, changed the time format on those pages to include seconds and milliseconds. [#104063][#104063]
+- Added a **Created SQL Connections** chart on the [**SQL**](../v23.1/ui-sql-dashboard.html) dashboard under **Metrics**. [#104072][#104072]
+- Added **Ranges in Catchup Mode** and **Rangefeed Catchup Scans Duration** charts to the [**Changefeeds**](../v23.1/ui-cdc-dashboard.html) dashboard. [#104122][#104122]
+- The sort setting on the [**Hot Ranges**](../v23.1/ui-hot-ranges-page.html) page is now persisted across page reloads and navigation. [#104112][#104112]
+- The [**Databases**](../v23.1/ui-databases-page.html) page now supports a large number of tables for a single database. If more than 40 tables are present in a database, the sort on the page will be disabled; however, it is still possible to filter by table name. [#103854][#103854]
+- On the **Statement Details** page, renamed the "Idle Latency" metric to "Client Wait Time" and separated it into its own chart. [#103915][#103915]
+- Added more search criteria options on the [**SQL Activity**](../v23.1/ui-overview.html#sql-activity) page:
+    - For the **Top** dropdown: `1000`, `5000`, and `10000`.
+    - For the **By** dropdown on the **Statements** tab: Last Execution Time, Max Latency, Max Memory, Min Latency, Network, P50 Latency, P90 Latency, Retries, and Rows Processed.
+    - For the **By** dropdown on the **Transactions** tab: Max Memory, Network, Retries, and Rows Processed. [#104054][#104054]
+- Added a new link to the **Range Status** page on the [**Replication**](../v23.1/ui-replication-dashboard.html) dashboard that opens the **Enqueue Ranges** page with the node ID already completed. [#104100][#104100]
+- The DB Console will no longer show `DROP INDEX` recommendations for unqiue indexes. [#104287][#104287]
```

Comment:
```suggestion
- The DB Console will no longer show `DROP INDEX` recommendations for unique indexes. [#104287][#104287]
```

Reply:
hmmm, weird a lot of these comments are outdated.... I have fixed these?

Reply:
Yeah I think you sent a commit when I was reviewing your last one so we crossed the streams. I've already approved.